odl/solvers/advanced/proximal_operators.py

```diff
@@ -846,6 +898,61 @@ def proximal_cconj_l1(space, lam=1, g=None, isotropic=False):
     prox_factory : function
         Factory for the proximal operator to be initialized
 
+    Notes
+    -----
+    The :math:`L_1`-norm :math:`F` is the functional
+
+        .. math::
+
+            F(x) = \\lambda ||x - g||_1.
+
+    The convex conjugate :math:`F^*` of :math:`F` is given by the indicator
+    function of the set :math:`box(\\lambda)`
+
+        .. math::
+
+            F^*(y) = \\lambda I_{box(\\lambda)} \left(
+            \left| \\frac{y}{\\lambda} \\right| +
+            \left\\langle \\frac{y}{\\lambda}, g \\right\\rangle
+            \\right)
```

Comment:
I will first give the same answer as I did somewhere else: I have simply typeset this `F^*(y) = lam ind_{box(lam)}(|y / lam| + <y / lam, g>)`

To answer the real question: we have an indicator function scaled by `lambda`. That scaling is pointless I guess, since that function only takes value 0 or infinity anyway. But I don't think it is necessarily that easy to simplify the argument of the indicator function... you can maybe try some completion of squares, but that I don't think it is very useful.